### PR TITLE
Compact metal cluster

### PR DESCRIPTION
**Pull request by varungur**
review /sdodson

### CODE REVIEW

File: ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml [hunk 1]
```diff
@@ -7458,3 +7458,142 @@ periodics:
...
+                memory: 4Gi
+              requests:
+                cpu: 100m
+                memory: 200Mi
+          tests:
+          - as: e2e-metal-upgrade
+            commands: TEST_SUITE=all run-upgrade-tests
+            openshift_installer:
+              cluster_profile: aws
+              upgrade: true
```
Comment (smarterclayton) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml:7544]: try testing with this set to false.

Comment (varungur) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml]: testing it now

Comment (varungur) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml:7544]: When I set the upgrade flag to false, it throws a different error. The error states that e2e-metal-upgrade is not available in the list

**Pull request closed without merging**